Configuration/DataProcessing/python/Impl/ppEra_Run2_2017_trackingOnly.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+"""
+_ppEra_Run2_2017_trackingOnly
+
+Scenario supporting proton collisions and tracking only reconstruction for HP beamspot
+
+"""
+
+import os
+import sys
+
+from Configuration.DataProcessing.Reco import Reco
+import FWCore.ParameterSet.Config as cms
+from Configuration.Eras.Era_Run2_2017_cff import Run2_2017
+
+from Configuration.DataProcessing.Impl.pp import pp
+
+class ppEra_Run2_2017_trackingOnly(pp):
+    def __init__(self):
+        pp.__init__(self)
+        # tracking only RECO is sufficient, to run high performance BS at PCL;
+        # some dedicated customization are required, though: customisePostEra_Run2_2017_trackingOnly
+        self.recoSeq=':reconstruction_trackingOnly'
+        self.cbSc=self.__class__.__name__
```

Comment:
this statement does not impact the pp class, nor do we want to modify such class.

We want with the statement
to assure that for the scenario we're defining here EI is not added to the sequence 
( https://github.com/cms-sw/cmssw/blob/master/Configuration/DataProcessing/python/Reco.py#L79 )